### PR TITLE
Highlights my name

### PR DESCRIPTION
**Pull request by Catalin12**

### CODE REVIEW

File: tasks/src/app/ordered-list/ordered-list.component.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<button (click)="changeListStatus()">{{ isListEnable ? "Disable" : "Enable" }}</button>
+<div *ngIf="isListEnable">
+	<ol>
+		<ng-container *ngFor="let name of listNames">
+			<li class="highlightingName" [ngClass]="{'highlightingName': name == 'Catalin'}"> {{name}} </li>
```
Comment (danielursu) [inline tasks/src/app/ordered-list/ordered-list.component.html]:
```suggestion
			<li [ngClass]="{'highlightingName': name === 'Catalin'}"> {{name}} </li>
```

**Review by danielursu — CHANGES_REQUESTED** [review]

File: tasks/src/app/ordered-list/ordered-list.component.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.highlightingName {
+	color: red;
```
Comment (danielursu) [inline tasks/src/app/ordered-list/ordered-list.component.css:2]: add font-weight

**Review by danielursu — CHANGES_REQUESTED** [review]

File: tasks/src/app/ordered-list/ordered-list.component.html (hunk 1 above)
Comment (danielursu) [reply tasks/src/app/ordered-list/ordered-list.component.html]: you want that highlightingName to appear only if it is your name, not for each item of the list

File: tasks/src/app/ordered-list/ordered-list.component.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.highlightingName {
```
Comment (danielursu) [inline tasks/src/app/ordered-list/ordered-list.component.css]:
```suggestion
.highlighting-name {
```
for css use kebab-case

**Review by danielursu — CHANGES_REQUESTED** [review]

File: tasks/src/app/ordered-list/ordered-list.component.html (hunk 1 above)
Comment (Catalin12) [reply tasks/src/app/ordered-list/ordered-list.component.html]: dubious but the "class" attribute was no longer taken into consideration when I used [ngClass] and everything went as required :))

**Review by danielursu — APPROVED** [review]

**Pull request merged**